packages/types/src/span.ts

```diff
@@ -234,6 +234,15 @@ export interface Span extends SpanContext {
    */
   instrumenter: Instrumenter;
 
+  /**
+   * Completion status of the Span.
+   *
+   * See: {@sentry/tracing SpanStatus} for possible values
+   *
+   * @deprecated Use `.setStatus` to set or update and `spanToJSON()` to read the status.
+   */
+  status?: string;
```

Comment:
Had to re-declare status on the `Span` interface because it's inherited from `SpanContext`. I don't think we want to deprecate it there in v7 but tbh not sure.

Side note:
This is similar to #10189 where I realized I also didn't deprecate `op` on the Span interface because of this inheritance yet. I'll tackle this in a follow-up PR though.

Reply:
yes, I think that's fine! I'd deprecate stuff on the `Span` interface directly and leave the span context alone, it will go away "automatically" when all methods that use it are gone in v8! 

And yeah, if we missed some there let's just add it, may very well be! 😅